core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.html

```diff
@@ -22,6 +22,8 @@
                 *ngIf="vm.showSummary"
                 [goals]="vm.experiment.goals"
                 [scheduling]="vm.experiment.scheduling"
+                [sessionsReached]="vm.results.sessions.total"
+                [suggestedWinner]="vm.winnerLegendSummary"
```

Comment:
Why not just pass the whole `vm`? too much?

Reply:
This will change shortly.